CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contribution Guidelines
+
+First of all, thanks for taking your time to contribute to the AliceNet development, you are awesome!
+
+## Submitting Issues
+
+Did you discover a bug? Do you have a feature request? Submitting issues is an easy way for you to contribute and help us to improve AliceNet. We use GitHub Issues to track all known bugs and feature requests.
+
+However, before logging an issue be sure to check the current opened and closed issues to see if someone already has submitted the same issue/request.
+
+See the [GitHub help guide](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue) for more information on submit a Github issue.
+
+## Vulnerabilities
+
+Please don't create issues for any security vulnerabilities. Instead, we would appreciate it if you reported them through our vulnerability disclosure form. This allows us to distribute a fix before the vulnerability is exploited.
+
+Additionally, if you believe that you've discovered a security vulnerability, you might qualify for our bug bounty program. Visit our [website](https://www.alice.net/) or our [discord server](https://discord.gg/bkhW2KUWDu) for details.
+
+## Contributions
+
+Before your submit your cool code, consider the following guidelines:
+
+Search on GitHub for an open or closed PR that relates to your submission. You don't want to duplicate existing efforts.
+
+Be sure that an issue describes the problem you're fixing, or documents the design for the feature you'd like to add. Discussing the design upfront helps to ensure that we're ready to accept your work.
+
+Also, read and follow our [Code of Conduct](CODE_OF_CONDUCT.md) to help us maintain the AliceNet development community a healthy environment.
+
+Finally, check the [Forking and submitting your development work](./docs/CREATING_PR.md) to get your work ready for submission.
```

Comment:
`docs/` should be move to wiki as we'll likely do a lot of swarming on it there.